sys/riscv/riscv/machdep.c

```diff
@@ -1150,8 +1158,12 @@ initriscv(struct riscv_bootparams *rvbp)
 	init_param2(physmem);
 	kdb_init();
 
-	if (boothowto & RB_VERBOSE)
+	if (boothowto & RB_VERBOSE) {
+		printf("FDT phys (%lx-%lx), kernel phys (%lx-%lx)\n",
+		    rvbp->dtbp_phys, rvbp->dtbp_phys + rvbp->dtbp_size,
+		    rvbp->kern_phys, rvbp->kern_phys + kernlen);
```

Comment:
Belongs in `fake_preload_metadata`.

Reply:
In that case it depends on EARLY_PRINTF. I guess that's fine since it's mostly for debugging.